diff --git a/target/linux/ath79/dts/ar9344_nec_wg600hp.dts b/target/linux/ath79/dts/ar9344_nec_wg600hp.dts
index e586dce..cec5f77 100644
--- a/target/linux/ath79/dts/ar9344_nec_wg600hp.dts
+++ b/target/linux/ath79/dts/ar9344_nec_wg600hp.dts
@@ -11,6 +11,6 @@
 	partition@40000 {
 		compatible = "denx,uimage";
 		label = "firmware";
-		reg = <0x040000 0x7c0000>;
+		reg = <0x040000 0xffc0000>;
 	};
 };
